packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Models/Types/EnumType.cs

```diff
@@ -1,72 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using System;
-using System.Collections.Generic;
-using Microsoft.CodeAnalysis;
-using Microsoft.Generator.CSharp.Input;
-
-namespace Microsoft.Generator.CSharp
-{
-    public class EnumType : TypeProvider
```

Comment:
this is renamed to `EnumTypeProvider` to align the name of models `ModelTypeProvider`.